pybossa/api/__init__.py

```diff
@@ -80,6 +80,10 @@
 from pybossa.auth.task import TaskAuth
 from pybossa.service_validators import ServiceValidators
 import requests
+from sqlalchemy.sql import text
+from pybossa.model.user import User
+from pybossa.core import db
+from pybossa.cache.task_browse_helpers import is_valid_searchable_column, get_searchable_columns
```

Comment:
is_valid_searchable_column is unused and can be removed

Reply:
great point. Thanks! I used it initially, but decided it was unnecessary